facilitator/Cargo.toml

```diff
@@ -17,9 +17,8 @@ hyper-rustls = "0.21.0"
 prio = "0.2"
 rand = "0.7"
 ring = { version = "0.16.15", features = ["std"] }
-rusoto_core = { version="0.45.0", default_features=false, features=["rustls"] }
-rusoto_mock = { version="0.45.0", default_features=false, features=["rustls"] }
-rusoto_s3 = { version="0.45.0", default_features=false, features=["rustls"] }
+rusoto_core = { version = "0.45.0", default_features=false, features=["rustls"] }
+rusoto_s3 = { version = "0.45.0", default_features=false, features=["rustls"] }
```

Comment:
You still missed spaces around the rest of the `=`s (for `default_features` and `features`, sorry)